examples/async_concurrent.rs

```diff
@@ -0,0 +1,45 @@
+use std::sync::Arc;
+
+use anyhow::{anyhow, Context as _};
+use open62541::{ua, AsyncClient};
+use open62541_sys::{UA_NS0ID_SERVER, UA_NS0ID_SERVER_SERVERSTATUS_CURRENTTIME};
+use tokio::task::JoinSet;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    env_logger::init();
+
+    let client =
+        Arc::new(AsyncClient::new("opc.tcp://opcuademo.sterfive.com:26543").context("connect")?);
+
+    let mut tasks = JoinSet::new();
+
+    // Spawn many tasks in order to trigger race conditions in multi-threaded use of open62541.
+    //
```

Comment:
This comment becomes obsolete as soon as the upstream issue has been fixed. Please add a link to the issue for context.

Reply:
Agreed. I assumed that "to trigger race conditions" refers to the bug.